src/features/files/components/LibraryImageCard.tsx

```diff
@@ -43,6 +56,18 @@ const LibraryImageCard: FC<LibraryImageCardProps> = ({
         >
           {imageFile.original_name}
         </Typography>
+        <Typography
+          alignSelf="center"
+          color="secondary"
+          component="span"
+          maxWidth="80%"
+          noWrap
+          overflow="hidden"
+          textOverflow="ellipsis"
+          variant="body2"
+        >
+          {dimensions.width} px x {dimensions.height} px
```

Comment:
I know this is how it was designed by @sefsh, but IMO the `'px x'` thing is difficult to read. I wonder if it would be better keeping `px` only at the end, e.g. `800 x 600 px`, and perhaps even replacing the `x` with some unicode character like `⨯`.

Reply:
I like that, got curious and checked what others do, and wikipedia does something like `800 × 600 pixels` which is a bit more explicit and `×`'s name is "MULTIPLICATION SIGN", which I guess is a step up if someone is reading it using a screen reader.